### PR TITLE
add barebone navbar

### PR DESCRIPTION
**Pull request by Claudwin**
Fixes[#3] (https://github.com/uc-blue-team/Chess-App/issues/3)

### CODE REVIEW

**Comment by liangma87** [comment]
@Claudwin I will take a look at it tomorrow!

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -9,7 +9,30 @@
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
-  <body>
+<body>
+  <div id="flex-content-container">
```
Comment (liangma87) [inline app/views/layouts/application.html.erb:13]: what is flex-content-container used for?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -9,7 +9,30 @@
...
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Login</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
     <%= yield %>
-  </body>
+</body>
```
Comment (liangma87) [inline app/views/layouts/application.html.erb:38]: I think you made a typo that div<flex-content-container> is not closed.

**Review by liangma87 — CHANGES_REQUESTED** [review]
Hi Claude, I provided some review comments and some questions for your PR.

Thanks,
Liang

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (Claudwin) [reply app/views/layouts/application.html.erb]: Flex is used in bootstrap to allow the nav to adjust to different screen sizes.

Comment (Claudwin) [reply app/views/layouts/application.html.erb:13]: > what is flex-content-container used for?

Flex is used in bootstrap to allow the nav to adjust to different screen sizes. @liangma87

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (Claudwin) [reply app/views/layouts/application.html.erb:38]: @liangma87 The code highlighted in red is deprecated and not included in the code. I just doubled check the request that line is not there.

Comment (liangma87) [reply app/views/layouts/application.html.erb:38]: @Claudwin sorry I thought I have typed it out but apparently I didn't.  I was trying to say div id="flex-content-container" is not closed.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (liangma87) [reply app/views/layouts/application.html.erb:13]: Thanks @Claudwin

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (Claudwin) [reply app/views/layouts/application.html.erb:38]: @liangma87 Thanks submitting a pull request now

Comment (Claudwin) [reply app/views/layouts/application.html.erb:38]: @liangma87 just closed the div can you review once again and merge if everything is good.

**Review by liangma87 — APPROVED** [review]
Thumbs up! good work!

**Pull request merged**